File: 0075-sort-colors/0075-sort-colors.py
class Solution:
    def sortColors(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        is_sorted = False
        
        while not is_sorted:
            is_sorted = True
            for i in range(len(nums) -1):
                if nums[i] > nums[i + 1]:
                    nums[i], nums[i + 1] = nums[i + 1], nums[i]
                    is_sorted = False

File: 0075-sort-colors/0075-sort-colors.js
/**
 * @param {number[]} nums
 * @return {void} Do not return anything, modify nums in-place instead.
 */
var sortColors = function(nums) {
    const counter = {0:0, 1:0, 2:0}
    
    for (const num of nums) {
        counter[num]++
    }
    
    let i = 0
    while (counter[0] > 0) {
        nums[i] = 0
        i++
        counter[0]--
    }
    while (counter[1] > 0) {
        nums[i] = 1
        i++
        counter[1]--
    }
    while (counter[2] > 0) {
        nums[i] = 2
        i++
        counter[2]--
    }
};